src/pages/club/InvitePage/InvitePage.tsx

```diff
@@ -0,0 +1,64 @@
+import ActiveButton from '@/components/ActiveButton/ActiveButton';
+import Avatar from '@/components/common/Avatar/Avatar';
+import BigLogo from '@/components/common/BigLogo/BigLogo';
+import ClubInfo from '@/components/common/ClubInfo/ClubInfo';
+import { LOGO_TEXT } from '@/constants/logo';
+import { PATH } from '@/constants/path';
+import useGetInviteClubInfoQuery from '@/hooks/query/club/useGetInviteClubInfoQuery';
+import useJoinClub from '@/hooks/query/club/useJoinClub';
+import { getStorage } from '@/utils/localStorage';
+
+import { useLocation, useNavigate, useParams } from 'react-router-dom';
+
+import {
+  ClubInfoWrapper,
+  InviteInfoWrapper,
+  LogoNInfoWrapper,
+  MessageStyled,
+  PageContainer,
+} from './InvitePage.style';
+
+const InvitePage = () => {
+  const { invitecode } = useParams();
+  const { pathname } = useLocation();
+  const { inviteClubInfo } = useGetInviteClubInfoQuery(invitecode ?? '');
+  const { joinClub, isLoading } = useJoinClub();
+  const navigate = useNavigate();
+  const isLogin = Boolean(getStorage('token'));
+
+  if (!inviteClubInfo) {
+    throw new Error('eventId is null');
+  }
+
+  const { name, info, logoImageUrl, memberCount } = inviteClubInfo;
+
+  const handleClickJoinButton = () => {
+    sessionStorage.setItem('returnpage', pathname);
+    navigate(PATH.LOGIN);
+  };
+
+  return (
+    <PageContainer>
+      <LogoNInfoWrapper>
+        <BigLogo logoText={LOGO_TEXT.SPACE_CLUB} />
+        <InviteInfoWrapper>
+          <ClubInfoWrapper>
+            <Avatar avatarSize="medium" profileImageSrc={logoImageUrl} isClub />
+            <ClubInfo reverse name={name} info={info} memberCount={memberCount} />
+          </ClubInfoWrapper>
+          <MessageStyled>{name} 클럽에 가입하시겠습니까?</MessageStyled>
+          <ActiveButton
+            buttonText="가입하기"
+            fontSize="smallTitle"
+            isLoading={isLoading}
+            onClick={() => {
+              isLogin ? joinClub(invitecode ?? '') : handleClickJoinButton();
```

Comment:
```suggestion
            onClick={handleClickJoinButton};
```
이렇게 하고 handleClickJoinButton 내부에서 isLogin여부를 판단해야 하지 않을까요?

Reply:
생각해보니 그렇네요! 감사합니다